app/serializers/submission_serializer/definitions/property.rb

```diff
@@ -17,7 +17,7 @@ def to_builder
           json.is_home_address is_home_address
           json.has_other_owners has_other_owners
           json.address address
-          json.property_owners Definitions::PropertyOwner.generate(property_owners)
+          json.property_owners Definitions::PropertyOwner.generate(property_owners.complete)
```

Comment:
Hm if a property owner is incomplete then I assumed the property would be marked as incomplete in the review application page and the user would need to remove or edit the other owner to submit 🤔

At the moment it looks like we are just presenting none so the user might think they have entered the details correctly as it's not being flagged 
<img width="791" alt="Screenshot 2024-03-21 at 12 22 30" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/51047911/da9591e9-aa24-4628-893c-9f689292ee00">